src/main/resources/i18n/Messages_en.properties

```diff
@@ -116,6 +116,11 @@ manifestCsvTopLevelNameNotUnique=Top Level Variable Name must be unique
 manifestCsvVariableNameNotUniqueWithinParent=No two children of the same parent can have the same name
 manifestCsvVariableParentDoesNotExist=Parent for variable does not exist
 manifestCsvWrongDataTypeForChild=Only tables and sections may be listed as parents
+#{0} is plot name
+monitoringPlotsNortheastCorner={0} Northeast corner
+monitoringPlotsNorthwestCorner={0} Northwest corner
+monitoringPlotsSoutheastCorner={0} Southeast corner
+monitoringPlotsSouthwestCorner={0} Southwest corner
```

Comment:
IMO we should either capitalize both the direction and the word "corner" or neither of them.

Reply:
This was done to follow naming of search columns. I am guessing we should update those to stay consistent too.

Reply:
I guess actually, since it follows the plot name, I will keep them lowercase here.